app/app_client.go

```diff
@@ -846,6 +846,17 @@ func (c *AppClient) OrganizationSetLogo(ctx context.Context, orgID string, logo
 	return err
 }
 
+// OrganizationGetLogo gets an organization's logo.
+func (c *AppClient) OrganizationGetLogo(ctx context.Context, orgID string) (string, error) {
```

Comment:
when does this get called? saw it set in [your pr](https://github.com/viamrobotics/rdk/pull/4615/files#diff-b1fa4d14d641cb4a96145ecbef6515d156304cc673295382b7284eaac324bf16) so added it but wasnt sure

Reply:
I think this is done for stubbing (i also wasnt sure but was just following the pattern)

Reply:
ok 🤷‍♀️